content/about/introduction/introduction.html

```diff
@@ -14,166 +14,161 @@
         <p>
-          The APG Task Force strives to make the APG welcoming to a broad audience by using plain and simple language wherever possible.
-          Consequently, even readers with minimal understanding of web development or accessibility can benefit from a significant portion of the guidance, e.g., the keyboard interaction patterns.
-          At the same time, many topics addressed by the APG are inherently complex and call for technically precise terminology.
-          Additionally, some important aspects of creating accessible experiences fall outside the scope of the APG.
-          This section describes the types of prerequisite knowledge and supporting resources that may enable readers to derive maximum benefit from the APG.
+          APGタスクフォースは、可能な限り平易でシンプルな言語を使用することで、広い視聴者にAPGを歓迎するよう努めています。
+          その結果、ウェブ開発やアクセシビリティの理解が最小限の読者でも、例えば、キーボードインタラクションパターンなど、ガイダンスのかなりの部分から利益を得ることができます。
+          同時に、APGによって対処される多くのトピックは本質的に複雑であり、技術的に正確な用語を必要とします。
+          さらに、アクセシブルな体験を作成するためのいくつかの重要な側面は、APGの範囲外にあります。
+          このセクションでは、APGから最大の利益を得るために読者が有する可能性のある前提知識の種類とサポートリソースを説明します。
         </p>
         <p>
-          Broadly speaking, the APG blends accessibility, UI design, and web engineering.
-          To successfully utilize the APG, it is not necessary for readers to have working knowledge in all three disciplines.
-          Accessibility specialists will find the APG more beneficial with at least a rudimentary understanding of HTML and CSS while designers and engineers will need familiarity with some accessibility fundamentals.
-          All readers will need a firm grasp of some basic ARIA concepts.
+          広く言えば、APGはアクセシビリティ、UIデザイン、およびウェブエンジニアリングを組み合わせています。
+          APGを成功裏に利用するためには、読者がこれら3つの分野すべてにおいて実務知識を持っている必要はありません。
+          アクセシビリティの専門家は、HTMLとCSSの基本的な理解があればAPGからより多くの利益を得るでしょうが、デザイナーやエンジニアはアクセシビリティの基本的な知識が必要になります。
+          すべての読者は、いくつかの基本的なARIAの概念をしっかりと把握する必要があります。
         </p>
         <p>
-          People who understand web design or engineering but are not familiar with accessibility and assistive technologies may be able to implement much of the guidance provided by the patterns and example code.
-          However, because many aspects of the guidance are neither prescriptive nor comprehensive, readers may need to make decisions that require assessing tradeoffs that could impact users with disabilities.
-          In addition to having some understanding of the needs of people with disabilities and the role of assistive technologies in satisfying them, it is essential that developers can perform some testing.
-          Relevant resources for developing an understanding of accessibility and assistive technologies include:
+          ウェブデザインまたはエンジニアリングを理解しているが、アクセシビリティや支援技術に慣れていない人々は、パターンと例のコードによって提供されるガイダンスの多くを実装できるかもしれません。
+          しかし、多くのガイダンスの側面が指示的でも包括的でもないため、読者は障害を持つユーザーに影響を与える可能性のあるトレードオフを評価する必要がある決定を下す必要があります。
+          障害を持つ人々のニーズと、それらを満たすための支援技術の役割を理解し、いくつかのテストを実行できることが不可欠です。
```

Comment:
some understanding -> ある程度理解していること
developers -> 開発者
としてそれぞれ訳してみました

```suggestion
          障害を持つ人々のニーズと、それらを満たすための支援技術の役割をある程度理解していることに加えて、開発者がいくつかのテストを実行できることが不可欠です。
```